README.md

```diff
@@ -8,85 +8,119 @@ Current state of the software:
-
-<!---
-<a href="https://www.codecogs.com/eqnedit.php?latex=y_1,&space;\ldots,&space;y_n&space;\sim&space;\int&space;k(\cdot&space;\mid&space;\theta)&space;P(d\theta)" target="_blank"><img src="https://latex.codecogs.com/gif.latex?y_1,&space;\ldots,&space;y_n&space;\sim&space;\int&space;k(\cdot&space;\mid&space;\theta)&space;P(d\theta)" title="y_1, \ldots, y_n \sim \int k(\cdot \mid \theta) \Pi(d\theta)" /></a>
-
-<a href="https://www.codecogs.com/eqnedit.php?latex=P&space;\sim&space;\Pi" target="_blank"><img src="https://latex.codecogs.com/gif.latex?P&space;\sim&space;\Pi" title="\Pi \sim P" /></a>
--->
+```math
+\begin{align*}
+  y_1, \dots, y_n &\sim \int k\left(\cdot \mid \theta\right) P\left(\text{d}\theta\right) \\[3pt]
+  P &\sim \Pi
+\end{align*}
+```
 
 For descriptions of the models supported in our library, discussion of software design, and examples, please refer to the following paper: https://arxiv.org/abs/2205.08144
 
+- Two lightweight interfaces are available for `Python` ([BayesMixPy](https://github.com/bayesmix-dev/bayesmix/tree/master/python#readme)) and `R` (BayesMixR)
+<!-- Add link after merge !!! -->
+
 # Installation
 
-## For end users
+Compiling and building `bayesmix` requires a modern C++ compiler, `git`, the GNU `make` utility, `cmake` (version >= 3.20) and `pkg-config`. Make these packages available varies between operating systems.
 
-**Prerequisites**: to build `bayesmix` you will need `git`, `pkg-config` and a recent version of `cmake`.
-On Linux machines, it is sufficient to run
+**Warning**: Check the version of `cmake` available in your package manager default repository. You may need to install a more recent version in another way. If this is your case, please go to the [official CMake GitHub repository](https://github.com/Kitware/CMake) and follow the installation instructions.
+
+## Requirements - Linux
+
+On Linux systems, the required packages can be installed via the system's default package manager. For instance, in Ubuntu or other Debian based distros it is sufficient to run
 
 ```shell
- sudo apt-get -y update && apt-get install -y
- sudo apt-get -y install git
- sudo apt-get -y install python3-pip
- sudo python3 -m pip install cmake
- sudo apt-get install -yq pkg-config
+sudo apt install git g++ make cmake pkg-config
 ```
 
-On macOS, after install HomeBrew, replace `sudo apt-get -y` with `brew`.
+## Requirements - MacOS
 
-To install and use `bayesmix`, please `cd` to the folder to which you wish to install it, and clone this repository with the following command-line instruction:
+On MacOS, the required packages can be installed via [Homebrew](https://brew.sh/). Once installed, it is sufficient to run
 
 ```shell
-git clone https://github.com/bayesmix-dev/bayesmix.git
+brew install git g++ make cmake pkg-config
 ```
 
-Then, by using `cd bayesmix`, you will enter the newly downloaded folder.
+## Requirements - Windows
```

Comment:
I think that this section is extremely long and takes the focus away from the tutorial which should be quite visible.

We could move all the installation instructions to separate markdown files or just the one for windows, I don't have a strong opinion on that.

Reply:
We can do an INSTALL.md, I see it's frequent practice and we also have a CONTRIBUTING.md already :)